include/wx/calctrl.h

```diff
@@ -26,12 +26,6 @@
 
 enum
 {
-    // show Sunday as the first day of the week (default)
-    wxCAL_SUNDAY_FIRST               = 0x0000,
-
-    // show Monday as the first day of the week
```

Comment:
Not sure why did the value of this one have to be changed, I'd leave it as is and just `SUNDAY_FIRST` to something non-zero.